src/scalper_utils.ts

```diff
@@ -93,8 +97,8 @@ export function orderSpliceMango(
   return spliceFactor;
 }
 
-// Splice delta hedge orders if available mango liquidity not supportive
-export function orderSpliceOpenBook(
+// Check if liquidity is supportive & splice order
```

Comment:
make the function name descriptive of what it is returning

Reply:
It does more than just splice now by checking how much liquidity there is we either go to Jupiter later or not

Reply:
better to have a long and bulky name then. Makes reading the code that calls it much easier to understand. Something like  liquidityCheckAndCountNumSplices